target/linux/ath79/image/common-mikrotik.mk

```diff
@@ -7,15 +7,14 @@ endef
 
 define Device/mikrotik_nor
   $(Device/mikrotik)
-  IMAGE/sysupgrade.bin := append-kernel | kernel2minor -s 1024 -e | \
-	pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | \
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
```

Comment:
Can you describe this change? I think the change below is enough

Reply:
kernel2minor is not needed when building sysupgrade image if yaffs2 support is present in kernel (which is true with this commit).